lib/twilio-ruby/framework/response.rb

```diff
@@ -0,0 +1,15 @@
+module Twilio
+  class Response
+    attr_accessor :status_code, :body
+
+    def initialize(status_code, body)
+      @status_code = status_code
+      body = '{}' if !body || body.empty?
```

Comment:
do these need parens? not familiar with ruby syntax.

Reply:
Yep.

Reply:
so it does or doesnt?

Reply:
Misread this syntax is fine.